src/main/java/org/jabref/gui/entryeditor/EntryEditor.java

```diff
@@ -143,6 +148,18 @@ public EntryEditor(LibraryTab libraryTab) {
 
         this.setOnDragDropped(event -> {
             BibEntry entry = this.getEntry();
+            if (event.getDragboard().hasContent(DragAndDropDataFormats.GROUP)) {
+               List<String> group = (List<String>) event.getDragboard().getContent(DragAndDropDataFormats.GROUP);
+               String changedGroup = null;
+                if (entry.getField(StandardField.GROUPS).isPresent()) {
+                    changedGroup = entry.getField(StandardField.GROUPS).get() + "," + group.get(0);
```

Comment:
1. This is more readable using `map` in optional, i.e., `changedGroup = entry.getField(...).map().orElse();
2. JabRef allows you to change the separator, see if you can use `preferences.getBibEntryPreferences().getKeywordSeparator()` or something similar instead of `", "

Reply:
Thank you, I will fix it. Now I have already got everything I want. if the checks are all right, I will post an official PR.